packages/lightning-lsp-common/src/__tests__/context.test.ts

```diff
@@ -205,6 +204,48 @@ it('configureSfdxProject()', async () => {
     expect(apexContents).not.toContain('declare type');
 });
 
+function verifyJsconfigCore(jsconfigPath: string) {
+    const jsconfigContent = fs.readFileSync(jsconfigPath, 'utf8');
+    expect(jsconfigContent).toContain('    "compilerOptions": {'); // check formatting
+    const jsconfig = JSON.parse(jsconfigContent);
+    expect(jsconfig.compilerOptions.experimentalDecorators).toBe(true);
+    expect(jsconfig.include[0]).toBe('**/*');
+    expect(jsconfig.include[1]).toBe('../../.vscode/typings/lwc/**/*.d.ts');
+    expect(jsconfig.typeAcquisition).toEqual({ include: ['jest'] });
+    fs.removeSync(jsconfigPath);
+}
+
+function verifyTypingsCore() {
+    const typingsPath = CORE_ALL_ROOT + '/.vscode/typings/lwc';
+    expect(typingsPath + '/engine.d.ts').toExist();
+    expect(typingsPath + '/lds.d.ts').toExist();
+    fs.removeSync(typingsPath);
+}
+
+function verifyCoreSettings(settings: any) {
+    expect(settings['files.watcherExclude']).toBeDefined();
+    expect(settings['eslint.nodePath']).toBeDefined();
+    expect(settings['perforce.client']).toBe('username-localhost-blt');
+    expect(settings['perforce.user']).toBe('username');
+    expect(settings['perforce.port']).toBe('ssl:host:port');
+}
+
+/*
+function verifyCodeWorkspace(path: string) {
```

Comment:
Is this not needed anymore ?

Reply:
I think we should remove this function, because from where it is called, it is commented. Or maybe let the function commented, so in a future if we need it, we can modify it.